src/Command/InfectionCommand.php

```diff
@@ -360,6 +360,16 @@ private function checkMetrics(): bool
             return false;
         }
 
+        foreach ([TestRunConstraintChecker::MSI_FAILURE, ''] as $type) {
```

Comment:
I'd prefer the second input to be `COVERED_MSI_FAILURE ` instead of an empty string.

Reply:
You're right. I guess, I saw this in another place in the infection code. Maybe we should change this everywhere it occures.